.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MetaMask/devs
+*                                   @MetaMask/shared-libraries-engineers
```

Comment:
I checked in with @mcmire on these and what we want is the newline char rather than a new line so I am just coming along behind with additional PRs to correct that rather than taking the suggestion. Thank you for calling this out!